quilt/tools/build.py

```diff
@@ -333,3 +334,17 @@ def generate_build_file(startpath, outfilename=DEFAULT_BUILDFILE):
     with open(buildfilepath, 'w') as outfile:
         yaml.dump(contents, outfile, default_flow_style=False)
     return buildfilepath
+
+def clone_git_repo(git_url, destination):
+    cmd = ['git', 'clone', '-q', '--depth=1', git_url, destination]
+    try:
+        proc = subprocess.Popen(cmd, cwd=os.getcwd(), env=os.environ.copy())
+    except Exception as exc:
+        print("Error %s while executing command %s", exc, cmd)
```

Comment:
You should also test the error handling :)

```
Error %s while executing command %s [WinError 2] File not found ['git', 'clone', '-q', '--depth=1', 'https://github.com/quiltdata/quilt-registry.git', 'tmp_git_clone']
```

Just raise a `CommandException` with the error message. It'll get printed out without the stack trace.